docs/faq.md

```diff
@@ -413,6 +413,34 @@ For reproducibility purposes, Jib sets the creation time of the container images
 jib.container.useCurrentTimestamp = true
 ```
 
+#### Please tell me more about reproducibility!
+
+_Reproducible_ means that building and rebuilding with Jib should
+produce an identical container image: the files should be the same,
+the file permissions should be the same, the file names should be
+the same, and the file creation and modification timestamps should
+be the same.
+
+Say that you ran `mvn jib:dockerBuild` with
+`<container><userCurrentTimestamp>true`. Jib will build an image
+whose creation time is the current timestamp. Now, a few seconds
+later, you run `mvn jib:dockerBuild` again without changing anything
+in your project. That is, the image created by the second `mvn
+jib:dockerBuild` build is identical to the first image except for
+the timestamp. But since the two images have different timestamps,
+the images will appear to be different to tools and infrastructure
+such Docker daemon, Docker Hub, registries, Kubernetes, etc). For
+example, now `docker images` will list two images, whereas without
+<useCurrentTimestamp>, the Docker daemon would have thought that
+the image already existed and resulted in no-op. Likewise, when you
+push the second image to a remote registry, the registry will treat
+it as a new image that doesn't exist. Tools will think it is a
+different image and download it again and cache it as a different
```

Comment:
As I said, this may give the impression that it will download an entire image again, while in reality I think it will be just a manifest. How about removing "download it again"?